### DISCUSSION TITLE
Extending CUDA Quantum with a new Simulator: hitting build problem

### DISCUSSION THREAD

**Discussion by stevescia** [NVIDIA/cuda-quantum#89] [Q&A]
while trying to run thru the mySimulator example, i am getting a buld error:

cmake .. -G Ninja -DNVQIR_DIR="$CUDA_QUANTUM_PATH/lib/cmake/nvqir"
-- Configuring done (0.0s)
CMake Error in CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "cudaq::cudaq-common".

CMake Error in CMakeLists.txt:
  IMPORTED_LOCATION not set for imported target "cudaq::cudaq-common".

==

i've checked my .cudaq directory and everythings seems to be installed including the cudaq-common lib:
/home/gitpod/.cudaq/lib/libcudaq-common.so

anybody know what i'm missing?

btw, CmakeLists.txt file contains same a doc example:

cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
project(DemoCreateNVQIRBackend VERSION 1.0.0 LANGUAGES CXX)
find_package(NVQIR REQUIRED)
nvqir_add_backend(MySimulator MySimulator.cpp)

**Comment by amccaskey** [comment]
Locally what does `echo $CUDA_QUANTUM_PATH` print for you?

**Reply by stevescia** [reply]
$ echo $CUDA_QUANTUM_PATH
/home/gitpod/.cudaq

here's a couple listings.. 

$ ls -lrt /home/gitpod/.cudaq/lib/cmake
total 12
drwxr-xr-x 2 root root  145 Apr 17 18:44 clang
drwxr-xr-x 2 root root 4096 Apr 17 18:44 mlir
drwxr-xr-x 2 root root 4096 Apr 17 18:44 llvm
drwxr-xr-x 2 root root  120 Apr 17 18:45 fmt
drwxr-xr-x 2 root root   91 Apr 17 18:45 nvqir
drwxr-xr-x 2 root root 4096 Apr 17 18:45 cudaq

ls -lrt /home/gitpod/.cudaq/lib/cmake/nvqir/
total 16
-rw-r--r-- 1 root root  823 Apr  9 12:34 NVQIRTargets-release.cmake
-rw-r--r-- 1 root root 5018 Apr  9 12:34 NVQIRTargets.cmake
-rw-r--r-- 1 root root 2095 Apr 16 16:27 NVQIRConfig.cmake
